src/track.js

```diff
@@ -170,8 +177,14 @@ class Track {
 	 * @return {Track}
 	 */
 	mergeSingleEvent(event) {
+		// There are no events yet, so just add it in.
+		if (!this.events.length) {
+			this.addEvent(event);
+			return;
+		}
+
```

Comment:
Hi @grimmdude im sorry to bring this up, may i ask you how this code comes into play?

Reply:
Hey @lplume, this bit has to do with merging single events into the event list.  Mostly applicable for when events are added using an explicit `startTick`.  If the newly added event is the only one in the list, then it doesn't need to be spliced into a particular location.

I can't remember exactly why I added it in this PR, but I think it may have been relevant to support the custom `EndTrackEvent` delta.